{
  try {
    assertTrue("Ready test failed",is.ready());
    is.read();
    assertTrue("More chars, but not ready",is.ready());
  }
 catch (  IOException e) {
    fail("Exception during ready test : " + e.getMessage());
  }
}
